build.gradle

```diff
@@ -44,7 +44,7 @@ ext {
 
     kotlinCommonStdlibModule="org.jetbrains.kotlin:kotlin-stdlib-common:${kotlinVersion}:sources"
     kotlinCompilerModule="org.jetbrains.kotlin:kotlin-compiler:${kotlinVersion}"
-    kotlinStdLibModule="org.jetbrains.kotlin:kotlin-stdlib:${kotlinVersion}"
```

Comment:
I suppose it should be `kotlinCommonStdlibModule` _instead of_ `kotlinStdLibModule`. The former is used to build native runtime and the latter is used as a dependency for compiler.